python/MANIFEST.in

```diff
@@ -14,13 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-global-exclude *.py[cod] __pycache__ .DS_Store
+# Reference: https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html
+
+graft pyspark
```

Comment:
@nchammas Seems like this ending up with adding all tests as well. Could we just include that json file alone?

Reply:
Yes, `graft` pulls everything.

We can try to just include what we think we need, but it's probably safer (and easier) in the long run to instead exclude what we don't want to package, like tests. Would that work for you?

Reply:
I agree that it's safer so we don't miss something out ... but let's just add `json` file alone .. I think it's more import to get rid of unrelated files ..

Reply:
OK. Are you planning to address this in #46331 (or some other PR), or would you like me to take care of it?

Reply:
I would appreciate if you make another PR :-)